src/gui/plugins/visualize_frustum/VisualizeFrustum.cc

```diff
@@ -0,0 +1,440 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
```

Comment:
```suggestion
 * Copyright (C) 2024 Open Source Robotics Foundation
```

Reply:
@ahcorde thank you for reviewing.
Before proceeding, I have a small query.
Do you want me to,
1. use `sign off and commit suggestion` for each review comment?
2. Or, do all the changes in my local branch and push it?

Additionally, for sign off, do I need to do to all the commits as I missed it in my first version, or commits from now onwards it's ok.
I'm bit confused.